django_project/cplus_api/models/layer.py

```diff
@@ -65,6 +65,10 @@ class LayerTypes(models.IntegerChoices):
     class Meta:
         abstract = True
 
+    def delete(self):
```

Comment:
@zamuzakki we use django-cleanup, so we don't need to manually the file. Could you please remove these lines?